llvm/test/Transforms/FunctionAttrs/read-write-errno.ll

```diff
@@ -0,0 +1,56 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 6
+; RUN: opt -passes=function-attrs -S < %s | FileCheck --check-prefixes=FNATTRS %s
+; RUN: opt -passes=attributor-light -S < %s | FileCheck --check-prefixes=ATTRIBUTOR %s
+
+define i32 @test_read_errno() {
+; FNATTRS-LABEL: define i32 @test_read_errno() {
```

Comment:
Looks like this isn't actually testing the inferred attributes. You need to pass `--function-attrs` or something like that to UTC.